lms/djangoapps/commerce/api/v1/tests/test_views.py

```diff
@@ -172,7 +172,7 @@ def test_update(self):
         self.assertIsNone(VerificationDeadline.deadline_for_course(self.course.id))
 
         # Generate the expected data
-        verification_deadline = datetime(year=2020, month=12, day=31, tzinfo=pytz.utc)
+        verification_deadline = datetime(year=2025, month=12, day=31, tzinfo=pytz.utc)
         expiration_datetime = datetime.now(pytz.utc)
```

Comment:
What is the goal here? To get the date from the future?

Reply:
If yes then get the current date and add a few years by utility methods of the DateTime module. Because these tests will fail again after 5 years. That's not a permanent solution.